ui.js

```diff
@@ -0,0 +1,53 @@
+import { fetchCompanies } from "./api";
+import {
+  ACCOUNT_EXECUTIVE_FIELD_NAME,
+  COMPANIES_TABLE_HEADERS,
+  COMPANY_NAME_FIELD_NAME,
+  CREATED_AT_FIELD_NAME,
+  REVENUE_YTD_FIELD_NAME,
+  STATUS_FIELD_NAME
+} from "./constants";
+
+export const makeTable = async () => {
+  const companies = await fetchCompanies();
+  // Print result of api call to the developer console
+  // Uncomment if you need it for debugging.
+  // While this method of logging variables of interest to the console is primitive, but often highly valuable debugging technique
+  // console.log(companies);
+
+  // Initialize new array and push a header row
+  const companiesToDisplay = [];
+  companiesToDisplay.push(COMPANIES_TABLE_HEADERS);
+
+
+  //Number separater with spaces
+  function numberWithSpaces(x) {
+    return x.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ' ');
+  }
+  // Here we simply rearrange company fields in the order in which we want to display them in UI
+  companies.map(company => {
+    const row = [];
+
+    row.push(
+      company[COMPANY_NAME_FIELD_NAME],
+      company[STATUS_FIELD_NAME],
+      new Date(company[CREATED_AT_FIELD_NAME]).toDateString(),

```

Comment:
This does not implement the behavior described in the issue. Please refer to the comments on the screenshot below:

![Uploading image.png…]()